tool/src/org/antlr/v4/codegen/model/ElementFrequenciesVisitor.java

```diff
@@ -134,6 +183,11 @@ protected void exitSubrule(GrammarAST tree) {
 			for (Map.Entry<String, MutableInt> entry : frequencies.peek().entrySet()) {
 				entry.getValue().v = 2;
 			}
+
+			int multiplier = tree.getType() == POSITIVE_CLOSURE ? 1 : 0;
```

Comment:
can we change this to a boolean rather than a multiplier? ;)  Ain't that fortran66 style hahah